src/pyelliptic/cipher.py

```diff
@@ -1,13 +1,18 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+"""
+src/pyelliptic/cipher.py
+========================
+"""
 
 #  Copyright (C) 2011 Yann GUIBET <yannguibet@gmail.com>
 #  See LICENSE for details.
 
 from openssl import OpenSSL
 
 
-class Cipher:
+# pylint: disable=redefined-builtin
+class Cipher:    # pylint: disable=old-style-class
```

Comment:
Can you make this a new style class and test if anything breaks? I think that's cleaner.

Reply:
Okay, will try and let you know.

Reply:
After making old-style class to new style class. All thinks works fine.Also test sending / receiving a message from a different computer.
So should i change old-style-class to new-style-class ?

Reply:
Yes please